content/departments/cloud/index.md

```diff
@@ -72,10 +72,6 @@ The Cloud team roadmap in available [here](https://docs.google.com/spreadsheets/
 - Support the initiative to [make the Cloud a preferred deployment method](https://docs.google.com/document/d/1IUFb2JkwqdLCf8B-FkiF40PyAFg50xfWnB6gQO2ViRQ/edit) from the platform and infrastructure perspective
 - [Cloud v2](https://docs.google.com/document/d/1GiOPJjuYrUahrZnENSLUCsujo2MCu2v_gw23SKNzE6E/edit#) - migrate current the Cloud (managed instances) from single-VM, Docker Compose based architecture to multi-node, GKE based architecture
 
-## Team
-
-{{generator:product_team.cloud}}
-
```

Comment:
Removed by mistake?

Reply:
this was actually a duplicate

it's already here

https://github.com/sourcegraph/handbook/blob/23f138d9891e1a5f34bd3c0bb2be467242a768b9/content/departments/cloud/index.md?plain=1#L1-L11

Reply:
Oh, good catch!